tests/test_enterprise/api/test_views.py

```diff
@@ -1121,6 +1121,7 @@ class TestEnterpriseCustomerListViews(BaseTestEnterpriseAPIViews):
                 'contact_email': 'fake@example.com',
                 'hide_course_original_price': False,
                 'sender_alias': 'Test Sender Alias',
+                'identity_providers': [],
```

Comment:
Might want to use some test data here

Reply:
Since this test case was expecting identity_provider as None so I added identity_providers with [], The test case with the value of identity_providers is at L1223. I have added it just to fix the old test case.